seed/seed.py

```diff
@@ -0,0 +1,25 @@
+import psycopg2
+
+conn = psycopg2.connect(
+    database="soengram",
+    user="postgres",
+    password="admin",
+    host="localhost",
+    port="5432"
+)
+
+cur = conn.cursor()
+
+cur.execute('''
+            CREATE TABLE "user" (
+                user_id SERIAL PRIMARY KEY,
+                email VARCHAR(255) UNIQUE NOT NULL,
+                username VARCHAR(255) UNIQUE NOT NULL,
+                full_name VARCHAR(255) NOT NULL,
+                password VARCHAR(255) NOT NULL,
```

Comment:
We should probably restrict password length to something much lower.

Reply:
password is hashed length

Reply:
My bad, completely forgot.